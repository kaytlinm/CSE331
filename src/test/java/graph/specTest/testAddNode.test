#Add Node Test File
#Tests that a graph can be created and a single node can be added to the graph

CreateGraph graph1
AddNode graph1 node1