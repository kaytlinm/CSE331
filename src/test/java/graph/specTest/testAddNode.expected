#Add Node Test File
#Tests that a graph can be created and a single node can be added to the graph

created graph graph1
added node node1 to graph1